ssg/build_remediations.py

```diff
@@ -166,22 +166,10 @@ def get_populate_replacement(remediation_type, text):
     sys.exit(1)
 
 
-def parse_from_file(file_path, env_yaml):
-    """
-    Parses a remediation from a file. As remediations contain jinja macros,
-    we need a env_yaml context to process these. In practice, no remediations
-    use jinja in the configuration, so for extracting only the configuration,
-    env_yaml can be an abritrary product.yml dictionary.
-
-    If the logic of configuration parsing changes significantly, please also
-    update ssg.fixes.parse_platform(...).
-    """
-
+def _parse_remediation(fix_file_lines):
     mod_file = []
```

Comment:
`mod_file` is a bad name. What comes to my first is that `remediation_content` is better.